src/ex1/script.js

```diff
@@ -0,0 +1,122 @@
+const addBtn = document.querySelectorAll('.add-task-btn');
+const clearAllBtn = document.querySelectorAll('.clear-all-btn');
+const addTaskField = document.querySelectorAll('.add-task-field');
+
+
+const taskList = [];
+addChillMsg();
+updateFooter();
+
+
+
+addBtn[0].addEventListener('click', () => {
+    addTaskFunc();
+});
+
+addTaskField[0].addEventListener("keydown", (event) => {
+    if (event.key === "Enter") {
+        addTaskFunc();
+    }
+});
+
+clearAllBtn[0].addEventListener('click', () => {
+    const tasks = document.querySelectorAll('.tasks');
+    tasks[0].replaceChildren();
+    while (taskList.length) taskList.pop();
+    tasks[0].append(ChillMsg());
+    updateFooter();
+});
+
+function addTaskFunc() {
```

Comment:
Personally i don't like using names like func in the func name, I think addTask is a great name :)